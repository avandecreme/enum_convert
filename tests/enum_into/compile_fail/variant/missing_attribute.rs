use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    // Missing #[enum_into(Target::Bar)]
    Foo,
}

enum Target {
    Bar,
}

fn main() {}
