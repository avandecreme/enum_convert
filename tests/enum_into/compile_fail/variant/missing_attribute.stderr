error[E0599]: no variant or associated item named `Foo` found for enum `Target` in the current scope
  --> tests/enum_into/compile_fail/variant/missing_attribute.rs:7:5
   |
4  |   #[enum_into(Target)]
   |  _____________-
5  | | enum Source {
6  | |     // Missing #[enum_into(Target::Bar)]
7  | |     Foo,
   | |    -^^^ variant or associated item not found in `Target`
   | |____|
   |
...
10 |   enum Target {
   |   ----------- variant or associated item `Foo` not found for this enum
